### PR TITLE
Update packages for AT 15.0

### PR DESCRIPTION
**Pull request by er-1**
This is related with #2093

### CODE REVIEW

**Comment by er-1** [comment]
On AT next x86_64, it failed when building `pveclib`:
```
In file included from /home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int512_ppc.h:27,
                 from /home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/vec_int512_runtime.c:23,
                 from /home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/vec_runtime_PWR10.c:27:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpequq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2617:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpeq’
 2617 |   return vec_cmpeq (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpgesq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2663:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpge’
 2663 |   return vec_cmpge (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpgeuq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2712:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpge’
 2712 |   return vec_cmpge (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpgtsq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2752:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpgt’
 2752 |   return vec_cmpgt (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpgtuq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2801:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpgt’
 2801 |   return vec_cmpgt (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmplesq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2841:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpge’
 2841 |   return vec_cmple (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpleuq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2890:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpge’
 2890 |   return vec_cmple (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpltsq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2931:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpgt’
 2931 |   return vec_cmplt (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpltuq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:2980:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpgt’
 2980 |   return vec_cmplt (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpneuq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3049:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_cmpeq’
 3049 |   return vec_cmpne (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpsq_all_eq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3097:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpeq_p’
 3097 |   return vec_all_eq (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpsq_all_ge’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3140:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpge_p’
 3140 |   return vec_all_ge (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpsq_all_gt’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3188:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpge_p’
 3188 |   return vec_all_ge (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpsq_all_le’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3236:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpge_p’
 3236 |   return vec_all_le (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpsq_all_lt’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3284:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpgt_p’
 3284 |   return vec_all_lt (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpsq_all_ne’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3334:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpne_p’
 3334 |   return vec_all_ne (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpuq_all_eq’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3379:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpeq_p’
 3379 |   return vec_all_eq (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpuq_all_ge’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3422:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpge_p’
 3422 |   return vec_all_ge (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpuq_all_gt’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3466:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpge_p’
 3466 |   return vec_all_ge (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpuq_all_le’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3510:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpge_p’
 3510 |   return vec_all_le (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpuq_all_lt’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3554:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpgt_p’
 3554 |   return vec_all_lt (vra, vrb);
      |   ^~~~~~
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h: In function ‘vec_cmpuq_all_ne’:
/home/tcbot/bot-worker/at/src/at16.0-0-alpha.debian-10_x86_64_ppc64le/sources/pveclib/src/pveclib/vec_int128_ppc.h:3600:3: error: invalid parameter combination for AltiVec intrinsic ‘__builtin_vec_vcmpne_p’
 3600 |   return vec_all_ne (vra, vrb);
      |   ^~~~~~
make[2]: *** [Makefile:1955: vec_dynrt_PWR10.lo] Error 1
```

**Comment by er-1** [comment]
I removed the update of `pveclib` on AT next. It'll be handled in another PR.

**Comment by mscastanho** [comment]
I submitted a bug report to pveclib in the meantime: open-power-sdk/pveclib#146

File: configs/15.0/packages/valgrind/sources [hunk 1]
```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=master
-ATSRC_PACKAGE_REV=93104368952c
+ATSRC_PACKAGE_VER=3.17.0
+ATSRC_PACKAGE_REV=dde556d51f82
```
Comment (mscastanho) [inline configs/15.0/packages/valgrind/sources:24]: @er-1  noticed there is no `VALGRIND_3_17_BRANCH` upstream and there is for the previous releases. Do you know why?

**Review by mscastanho — APPROVED** [review]
LGTM. I just have 1 small question.

**Comment by mscastanho** [comment]
And also, this may need a rebase to fix conflicts.

File: configs/15.0/packages/valgrind/sources (hunk 1 above)
Comment (er-1) [reply configs/15.0/packages/valgrind/sources:24]: @mscastanho no idea
There is a `VALGRIND_3_16_BRANCH` but no `VALGRIND_3_15_BRANCH`. May be it's something with odd versions.

**Comment by er-1** [comment]
> And also, this may need a rebase to fix conflicts.

Done

**Review by tuliom — APPROVED** [review]

**Review by mscastanho — APPROVED** [review]

**Pull request merged**